### PR TITLE
dd zope.proxy

### PR DESCRIPTION
**Pull request by ChrisBarker-NOAA**
@conda-forge/help-python: this one has been weird, I can only get it to work with python3.8 on my machine, not python2.7 --hopefully it will work on the CI.

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

@daf: you may need to post a comment on this one.

### CODE REVIEW

File: recipes/zope.proxy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  imports:
+    - zope.proxy
+
+about:
+  home: https://github.com/zopefoundation/zope.proxy
+  license: ZPL 2.1
+  license_family: OTHER
+  license_file: LICENSE.txt
+  summary: 'special objects which serve as mostly-transparent wrappers around another object'
+
```
Comment (npavlovikj) [inline recipes/zope.proxy/meta.yaml]: Please remove all comments and empty line here.

**Comment by npavlovikj** [comment]
@daf, are you agreeing to be a maintainer on this recipe?

**Comment by PythonCHB** [comment]
With only the AppVeyor failure, we can call this good to go?

**Comment by npavlovikj** [comment]
The recipe LGTM - I am just waiting on the other maintainer to say whether he is ok being listed here.

**Comment by daf** [comment]
@npavlovikj yup, I'm good to help maintain.

**Comment by npavlovikj** [comment]
Thank you for contributing, @ChrisBarker-NOAA !

**Comment by ChrisBarker-NOAA** [comment]
Thanks -- prepare for more zope.* pacakges -- this was one of the trickier ones (compiled code), now I should be able to bang out the rest.

**Pull request merged**